### ISSUE TITLE
Bad diff colors

### ISSUE THREAD

**Issue by busslina** [Kos-M/GogsThemes#15]
I just installed this theme (dark) and it's beautiful. However diff section fails IMO:

![imagen](https://github.com/Kos-M/GogsThemes/assets/73592852/3b205584-bd07-4ee2-b6f8-799313e3a310)

**Comment by Kos-M** [comment]
Thanks for bring this to our attention. Indeed there are some sections that needs improvements , including this.
I'm not using Gogs currently , if you notice anything not feels right , feel free to file a new issue.

**Comment by busslina** [comment]
May I ask if you are using an alternative to Gogs?

**Comment by Kos-M** [comment]
Im using , Gitea  these days..

Note if you re planing to use it in a raspbery pi better use SQLite  as database, its more safe for data . Had some data corruptions with gogs and mariadb.

**Comment by busslina** [comment]
> Im using , Gitea  these days..
> 
> Note if you re planing to use it in a raspbery pi better use SQLite  as database, its more safe for data . Had some data corruptions with gogs and mariadb.

I'm using Raspberry Pi 3 + Postgresql. I think it must be more secure than MariaDb.

Are you happy with Gitea? I discarded due a video explaining it's only open source in its name.

**Comment by Kos-M** [comment]
Yes , im using it about one year , its stable and lightweight ( sqlite version at least) 
I think its ok to have a fine-grained version , to make profit of it.
Look GitLab , offers a self hosted version aka community , along with  its premium version..
